src/BcPrebidVast.js

```diff
@@ -43,19 +43,34 @@ function doPrebid(options, callback) {
 			}
 
 			if (options.prebidConfigOptions) {
-				// DFP reqired prebid cache
-				if (!options.enablePrebidCache && options.prebidConfigOptions.cache && !options.dfpParameters) {
-					delete options.prebidConfigOptions.cache;
-				}
+                // Enable Prebid Cache by default
+			    if (options.enablePrebidCache !== false) {
+                    options.enablePrebidCache = true;
+                    // If no Prebid Cache url is set, use AppNexus' Prebid Cache by default
+                    if (!options.prebidConfigOptions.cache) {
+                        options.prebidConfigOptions.cache = {};
+                    }
+                    if (!options.prebidConfigOptions.cache.url) {
+                        var defaultCacheURL = 'https://prebid.adnxs.com/pbc/v1/cache';
+                        options.prebidConfigOptions.cache.url = defaultCacheURL;
+                        _logger.log(_prefix, 'No Prebid Cache url set - using default: ' + defaultCacheURL);
+                    }
+				} else {
+                    // DFP requires Prebid Cache, but otherwise remove the unused cache object if present
+                    if (options.prebidConfigOptions.cache && !options.dfpParameters) {
+                        delete options.prebidConfigOptions.cache;
+                    }
+                }
 				$$PREBID_GLOBAL$$.bc_pbjs.setConfig(options.prebidConfigOptions);
 			}
 
 			// activate prebid cache (DFP reqired prebid cache)
-			if (options.enablePrebidCache || options.dfpParameters) {
-				$$PREBID_GLOBAL$$.bc_pbjs.setConfig({
-					usePrebidCache: true
-				});
-			}
+            // ** NOTE ** 9/6/18: { usePrebidCache:true } has apparently been deprecated from the Prebid.js API
```

Comment:
Matt asked us not keep the commented code. Please remove commented code related deprecated setting. Vikki going to add information about usePrebidCache setting to release note.

Reply:
Done.